### DISCUSSION TITLE
Custom human readable text support

### DISCUSSION THREAD

**Discussion by vresetnikov** [metafloor/bwip-js#288] [Q&A] [answered]
Hello,

Does the library allow setting a custom human readable text under the barcode? I cannot find the option to do so.

**Comment by metafloor** [comment, marked as answer]
The option is `alttext`.  For example:

http://bwipjs-api.metafloor.com/?bcid=code128&text=0123456789&alttext=Hello%20World&includetext

**Reply by vresetnikov** [reply]
Thank you
